nf/modules/local/gatk/intervallisttools/main.nf

```diff
@@ -0,0 +1,27 @@
+process GATK_INTERVALLISTOOLS {
+    label 'C4'
+    container "pgc-images.sbgenomics.com/d3b-bixu/gatk:4.2.0.0R"
+    input:
+    path(input)
+    path(input2)
+    val(action)
+
+    output:
+    path('*interval_list'), emit: output_intervallist
+
+    script:
+    def ext_args = task.ext.args ?: ''
+    """
+    /gatk IntervalListTools \\
+    -I $input \\
+    --SECOND_INPUT $input2 \\
+    --ACTION $action \\
+    $ext_args
```

Comment:
I feel like modules should be able to operate without a config. This module won't work unless the user adds an `-O` argument to the config. You should have something like `-O ${prefix}.interval_list` where prefix gets a default value but can be set in the config.